### PR TITLE
chore(@toss/error-boundary): change `useIsMounted` dependencies local to `@toss/react`

### PR DESCRIPTION
**Pull request by minsoo-web**
## Overview

I change `useIsMounted` hooks import local to package (`@toss/react`).
Because, `useIsMounted` is already exist on `@toss/react`.

## PR Checklist

- [x] I read and included theses actions below

1. I have read the [Contributing Guide](https://github.com/toss/slash/blob/main/.github/CONTRIBUTING.md)
2. I have written documents and tests, if needed.

### CODE REVIEW

File: packages/react/error-boundary/package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
+    "@toss/react": "workspace:^1.4.0",
```
Comment (ChanhyukPark-Tech) [inline packages/react/error-boundary/package.json:24]: `@toss/react`'s stable version is 1.5.0 now 
Is there a reason why you installed 1.4.0?

Comment (minsoo-web) [reply packages/react/error-boundary/package.json:24]: Oh, I didn't check `@toss/react`'s stable version.
I will change this. Thanks 🙇‍♂️

**Comment by manudeli** [comment]
Could you guys review https://github.com/suspensive/react/pull/92 ? Maybe this Pull Request will be affected I think.

[When first I add this ErrorBoundaryGroup in @toss/error-boundary](https://github.com/toss/slash/pull/157), I couldn't consider useIsMounted's rerendering side effect. after @suspensive/react changing, I'd like to add this fixing in ErrorBoundaryGroup of @toss/error-boundary too.

**Review by ChanhyukPark-Tech — APPROVED** [review]
thanks!😄

**Comment by ChanhyukPark-Tech** [comment]
>https://github.com/toss/slash/pull/157, I couldn't consider useIsMounted's rerendering side effect. after @suspensive/react changing, I'd like to add this fixing in ErrorBoundaryGroup of @toss/error-boundary too.

@manudeli 

I understand where you're worried.
There is no direct connection with this PR, so this PR will be merged.

**Pull request merged**